stdlib/lib2to3/pgen2/tokenize.pyi

```diff
@@ -1,6 +1,40 @@
-from lib2to3.pgen2.token import *  # noqa
+import sys
+from lib2to3.pgen2.token import *
 from typing import Callable, Iterable, Iterator
 
+# fmt: off
```

Comment:
Desperate times call for desperate measures.

Reply:
Just let Black format it.